provision-osx/tasks/install_homebrew.yml

```diff
@@ -0,0 +1,85 @@
+---
+
+- 
+  name: Check homebrew directory permissions
+  file:
+    path: "{{ homebrew_prefix }}"
+    owner: root
+    group: admin
+    state: directory
+    mode: 0775
+  become: yes
+
+- 
+  name: Check homebrew directory exists
+  file:
+    path: "{{ homebrew_install_path }}"
+    owner: "{{ ansible_user_id }}"
+    group: admin
+    state: directory
+    mode: 0775
+  become: yes
+
+- 
+  name: Install homebrew
+  git:
+    repo: "{{ homebrew_repo }}"
+    version: master
+    dest: "{{ homebrew_install_path }}"
+    update: no
+    accept_hostkey: yes
+    depth: 1
+
+- 
+  name: Check permissions on homebrew_brew_bin_path
+  file:
+    path: "{{ homebrew_brew_bin_path }}"
+    state: directory
+    owner: "{{ ansible_user_id }}"
+    group: admin
+    mode: 0775
+  become: yes
+
+- 
+  name: Check permissions on homebrew_install_path
+  file:
+    path: "{{ homebrew_install_path }}"
+    state: directory
+    owner: "{{ ansible_user_id }}"
+    group: admin
+    recurse: true
+  become: yes
+
+- name: Check if homebrew binary is already in place.
+  stat: "path={{ homebrew_brew_bin_path }}/brew"
+  register: homebrew_binary
+
+- 
+  name: Symlink brew
+  file:
+    src: "{{ homebrew_install_path }}/bin/brew"
+    dest: "{{ homebrew_brew_bin_path }}/brew"
+    state: link
+  when: homebrew_binary.stat.exists == false
+  become: yes
+
+- 
+  name: Brew update
+  command: "{{ homebrew_brew_bin_path }}/brew update --force"
+  when: homebrew_binary.stat.exists == false
+
+- 
+  name: Ensure taps are added
+  homebrew_tap: "tap={{ item }} state=present"
+  with_items: "{{ homebrew_taps }}"
```

Comment:
Yes it should, underneath it's just using brew tap which supports the env vars.

Is there any way of setting these environment variables for all tasks in one location?